### ISSUE TITLE
Add command to open Pear from CLI similar to VSCode

### ISSUE THREAD

**Issue by Fryingpannn** [trypear/pearai-app#45]
Similar to how with VSCode you can open the current folder in CLI with the command `code .`, we should be able to open Pear in the current directory with the command `pear .`. Or simply open Pear with command `pear`.

Edit: this works on mac already, but needed for Windows.

**Comment by yawningwinner** [comment]
is this issue still open?

**Comment by timknappe** [comment]
Yes this is still open, current convention on windows was for base VS to add it in the installer. Since this is likely to be included in some walktroughs, this would require a native implementation long term.

**Comment by maskedwolf4** [comment]
Hello @Fryingpannn is the issue solved ?? if not I would like to contribute to it, can you please assign it to me and help me in setup process??
